pkg/manifest/installer.go

```diff
@@ -408,10 +443,20 @@ func cRDKindObjects(objects object.K8sObjects) object.K8sObjects {
 	return ret
 }
 
-func nonCRDKindObjects(objects object.K8sObjects) object.K8sObjects {
+func nsKindObjects(objects object.K8sObjects) object.K8sObjects {
+	var ret object.K8sObjects
+	for _, o := range objects {
+		if o.Kind == "Namespace" {
+			ret = append(ret, o)
+		}
+	}
+	return ret
+}
+
+func nonNsCRDKindObjects(objects object.K8sObjects) object.K8sObjects {
```

Comment:
Leave a TODO here to refactor the code so that we are checking for objects not being in a map, rather than relying on not being a specific type.

Reply:
TODO added.

Reply:
Actually I added the objectsNotInLists() func.